lib/validate/static_utils.rb

```diff
@@ -29,13 +38,17 @@ def style_tf
 
 def format_tf
   # Apply the canonical format and style on current working folder.
-  print "INFO: Formatting Terraform configurations...\n".yellow  
-  message = `terraform fmt -diff=true 2>&1`
+  print "INFO: Formatting Terraform configurations...\n".yellow
+  if ENV['TERRAFORM_VERSION'].start_with?("0.12")
+    message = `terraform fmt -diff 2>&1`
+  elsif ENV['TERRAFORM_VERSION'].start_with?("0.11")
+    message = `terraform fmt -diff=true 2>&1`
+  end
 
   # Check the styling message.
   if not message.empty?
     raise "ERROR: Formatting Terraform configurations failed!\n#{message}\n".red
   else
     print "INFO: Done!\n".green
   end
-end
+end
```

Comment:
Can we add a new line ?

Reply:
Sure, but not clear on where you want the new line. Can you clarify? Thanks,

Reply:
Can we a newline at the end of this file ?

Reply:
Done c419bfc